<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterDeviceInterface</class>
 <widget class="QDialog" name="MasterDeviceInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Master Device Window</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>173</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>MODE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_RobotControllerMode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>READY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_Ready"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_ConnectRobot">
        <property name="text">
         <string>Connect Robot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_DisconnectRobot">
        <property name="text">
         <string>Disconnect Robot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_StopTeleop">
        <property name="text">
         <string>STOP TELEOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_StartTeleop">
        <property name="text">
         <string>START TELEOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>174</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>q (deg)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_JPos"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>216</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>EPOS3 </string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>176</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_EPOS3Status"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_resetEPOS3_Fault">
       <property name="text">
        <string>RESET FAULT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_QuitM">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
